library/src/main/scala/scala/scalajs/runtime/UndefinedBehaviorError.scala

```diff
@@ -19,8 +19,7 @@ class UndefinedBehaviorError(message: String, cause: Throwable)
   def this(cause: Throwable) =
     this(
         "An undefined behavior was detected" +
-        (if (cause == null) "" else ": " + cause.getMessage),
-        cause)
+        (if (cause == null) "" else ": " + cause.getMessage), cause)
```

Comment:
No, this is an undesirable (but tolerated) change. An argument spanning two lines or more should not share one of its lines with another argument. This is filed as https://github.com/olafurpg/scalafmt/issues/502